webapp/settings.py

```diff
@@ -25,7 +25,7 @@
 SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
@yt It should be configurable via .env

Reply:
It is, at the end of settings file if environment is local, the settings gets overridden by settings_development.py file.